creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a feature that allows your payments to be sent directly to your account while Moongate handles the associated fees.
+>
+> Imagine you are an event organizer using Moongate to host and manage your events. You want to sell tickets online and receive payments from attendees. Instead of setting up your own payment processing system, you can use Stripe Connect.
+>
+> **Moongate:** Acts as the ticketing platform and integrates with Stripe Connect to facilitate transactions.
+>
+> **Organizer (You):** Receives payments from ticket sales directly to your Stripe account, eliminating the need to request payouts from Moongate. This allows you to focus on organizing your event while Stripe Connect handles the payment processing.
+>
+> **Platform Fees:** Moongate receives the platform fees automatically, so you don’t have to worry about deducting and handling these fees yourself.
```

Comment:
Organizer does not handle deduction of fees in traditional method also, so we can eliminate the reason